spec/integration/integration_helper.rb

```diff
@@ -5,6 +5,7 @@
 begin
   require 'active_support'
 rescue LoadError
+  print 'No "active_support" loaded'
```

Comment:
any clue why do we rescue this error at all? does things still work without Active support?